tests/utils.go

```diff
@@ -1022,6 +1030,14 @@ func NewRandomVMIWithEphemeralDisk(containerImage string) *v1.VirtualMachineInst
 	return vmi
 }
 
+func NewRandomLongNameVMIWithEphemeralDisk(containerImage string) *v1.VirtualMachineInstance {
```

Comment:
I doubt that we need such VMIs very often in tests. I would not create helpers and directly do that in the tests. It would be strange if we would create helper functions in `utils.go` for any test case scenario which we have.

Do something like 

```golang
	vmi := NewRandomVMIWithEphemeralDisk(RegistryDiskFor(RegistryDiskAlpine))
        vmi.Name = "testvmi"+rand.String(63)
```

in the test.

Reply:
That's a valid point. Thanks for the suggestion.